tests/functions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function isPartialRun(): bool
+{
+    foreach ($_SERVER['argv'] as $arg) {
+        if (str_starts_with($arg, '--filter')) {
```

Comment:
used `str_starts_with` to support the case of `--filter=...` instead of `--filter ...`